Index.html

```diff
@@ -0,0 +1,282 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="Style.css" />
+    <title>Portfolio</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <section class="mobile-container">
+        <header class="topnav">
+          <nav>
+            <a href="#home" class="active">My Logo</a>
+            <a class="icon"><img src="images/btn.png" /></a>
+          </nav>
+        </header>
+        <main class="first-section">
+          <h1 class="headline-1">Hey There. I’m Yaser.</h1>
+          <h1 class="title-1">I'm a Software Developer</h1>
```

Comment:
- [ ] `h1` tag is for the main title `Hey There, I'm Yaser.` And `h1` tag only appears one time per webpage. It would be better to use other `hx` tags and maintain the hierarchy.  Kindly fix this issue.